src/Security/Authentication/test/OpenIdConnect/OpenIdConnectConfigurationTests.cs

```diff
@@ -28,6 +29,7 @@ private void ConfigureDefaults(OpenIdConnectOptions o)
     public async Task CanForwardDefault()
     {
         var services = new ServiceCollection().AddLogging();
+        services.AddSingleton<IConfiguration>(new ConfigurationManager());
```

Comment:
We should add a new shared test helper method to centralize this, some kind of BuildAuthServices(Action<IServiceCollection> action) to roll the AddLogging/ConfigurationManager/AddAuthentication calls for all these tests